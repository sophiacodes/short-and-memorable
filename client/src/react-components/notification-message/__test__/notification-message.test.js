import React from 'react';
import NotificationMessage from '../';
import renderer from 'react-test-renderer';

it('renders <NotificationMessage /> correctly', () => {
  const tree = renderer
    .create(<NotificationMessage className="error">This is a notification message</NotificationMessage>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});