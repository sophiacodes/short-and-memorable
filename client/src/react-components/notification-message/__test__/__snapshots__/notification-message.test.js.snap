// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <NotificationMessage /> correctly 1`] = `
<p
  className="notification-message error"
>
  This is a notification message
</p>
`;
